VultisigApp/VultisigApp/Views/Swap/SwapTransaction.swift

```diff
@@ -58,6 +58,30 @@ class SwapTransaction: ObservableObject {
     var inboundFeeDecimal: Decimal? {
         return quote?.inboundFeeDecimal(toCoin: toCoin)
     }
+    
+    var gasInReadable: String {
+        guard var decimals = Int(fromCoin.decimals) else {
+            return .empty
+        }
+        if fromCoin.chain.chainType == .EVM {
+            // convert to Gwei , show as Gwei for EVM chain only
+            guard let weiPerGWeiDecimal = Decimal(string: EVMHelper.weiPerGWei.description) else {
+                return .empty
+            }
+            return "\(Decimal(gas) / weiPerGWeiDecimal) \(fromCoin.feeUnit)"
```

Comment:
Gas for SwapTransaction represents total fee needed (GasLimit * gasPrice). 

To get gas you looking for we should fetch ChainSpecific from BlockChainService and get gas property 

```
let chainSpecific = try await blockchainService.fetchSpecific(
    for: tx.fromCoin,
    action: .swap,
    sendMaxAmount: false
)
```

Reply:
<img width="878" alt="image" src="https://github.com/vultisig/vultisig-ios/assets/749608/77b15b31-4bac-499f-a810-46f5070258cb">
Check the gas , that's not correct